helm/reana/templates/reana-workflow-controller.yaml

```diff
@@ -248,3 +263,11 @@ spec:
           path: {{ $workspace_path._0 }}
       {{- end }}
       {{- end }}
+      {{- if .Values.workspaces }}{{ if .Values.workspaces.paths }}
+      {{- range $workspace_paths := .Values.workspaces.paths }}
+      - name: {{ $workspace_paths | quote | replace "/" "" }}
+        hostPath:
+          path: {{ $workspace_paths }}
+      {{- end }}
+      {{- end }}{{ end }}
+      
```

Comment:
Extremely minor: there are some spaces in the last line, that's why it still shows `No new line` error icon